_inc/client/components/jetpack-dialogue/index.jsx

```diff
@@ -17,13 +17,31 @@ import onKeyDownCallback from 'utils/onkeydown-callback';
 import { imagePath } from 'constants/urls';
 
 class JetpackDialogue extends Component {
+	clickBackground( e ) {
+		this.props.dismiss( e );
+	}
+
+	// capture the ESC key globally
+	componentDidMount(){
+		document.addEventListener('keydown', this.props.dismiss, false);
```

Comment:
I think these new rules should respect `this.props.showDismiss` -- there may be instances we don't want to allow dismissal without action.

Reply:
Good idea - fixed

Reply:
`git push`? While you're in there can you apply code standard spacing?

Reply:
Fixed